Module.yaml

```diff
@@ -0,0 +1,54 @@
+version: 1.0.0
+name: CSVEgress
+description: Module collects incoming data and stores it to CSV file that is then transfered further to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_CSV_Output.png
+type: output
+tags:
+  - csv
+  - egress
+  - input
+categories:
+  - id: category
+    name: Egress
```

Comment:
Please clarify with Christos if it is output or processing module. If output then it cannot pass data to another module, as it will be the last module in a pipeline.
If processing then clarify passing the file to the next module, because other modules do not process files, they process json.

Reply:
it's updated to process now.